xla/service/cpu/onednn_ops_rewriter.cc

```diff
@@ -423,17 +429,12 @@ class OneDnnOpsRewriterVisitor : public DfsHloRewriteVisitor {
     return OkStatus();
   }
 
-  Status HandleConvert(HloInstruction* instr) override {
-    HloInstruction* ln_instr;
-    HloInstruction* convert_instr;
```

Comment:
`convert_instr` is still used in lines 442-443 (of the latest commit).

[XLA Linux CPU CI](https://source.cloud.google.com/results/invocations/f288bd54-030a-4d60-b0d4-dc90067c8f7d) error message:
```
external/xla/xla/service/cpu/onednn_ops_rewriter.cc:442:12: error: use of undeclared identifier 'convert_instr'
  442 |           (convert_instr->shape().element_type() == PrimitiveType::BF16) ||
      |            ^
external/xla/xla/service/cpu/onednn_ops_rewriter.cc:443:12: error: use of undeclared identifier 'convert_instr'
  443 |           (convert_instr->shape().element_type() == PrimitiveType::F16);
      |            ^
11 warnings and 2 errors generated.
```

Reply:
Sorry about that...I missed it while merging main to resolve conflicts on my branch.